tf2onnx/onnx_opset/tensor.py

```diff
@@ -1248,3 +1248,142 @@ def version_10(cls, ctx, node, **kwargs):
         target_dtype = TensorProto.INT64
         if seq_len_dtype != target_dtype:
             ctx.insert_new_node_on_input(node, "Cast", node.input[1], to=target_dtype)
+
+
+@tf_op("ReverseV2")
+class ReverseV2:
+    @classmethod
+    def version_9(cls, ctx, node, **kwargs):
+        raise NotImplementedError("ReverseV2 is not supported to convert in OPSET 9,"
+                                  " if possible please try using OPSET>=10 instead.")
+
+    @classmethod
+    def version_10(cls, ctx, node, **kwargs):
+
+        axes_node = node.inputs[1]
+        axes = axes_node.get_tensor_value() # Is a Python list.
```

Comment:
axes is always constant in tf?

Reply:
I think so. Is it not?

Reply:
in tf, the op can accept tensor as input.
but it's fine the conversion function only support const, while it will be better you put a checker in the beginning of function, so the reader know the limitation of your function quickly.

Reply:
This is confusing for me. Axes can be a tensor?

Reply:
K nvm. I just read the documentation. Missed it before. Thanks for pointing that out.